### DISCUSSION TITLE
Exporting as PDF

### DISCUSSION THREAD

**Discussion by abdosaeedelhassan** [rappasoft/laravel-livewire-tables#327] [Q&A]
i already used ($this->columns() and $this->rows) to get all data, i know that is the column is hidden, but in the datatable i need to make some columns not exported to the pdf (like previews version of this package), how can i get that?

**Comment by rappasoft** [comment]
The bulk only gives you a list of ID's, so just get the data you need to export and not the data you don't need?

**Reply by abdosaeedelhassan** [reply]
Yes, it's worked for me, but i want to do it in global, one export code worked for all datatables, i get all needed, but i can't determine if column is not to be exported to the pdf, i did it manually, but I want to ensure the continuity of using this package

**Comment by lrljoe** [comment]
Please see docs on Bulk Actions, possible in v2+
